test/passing/let_binding.ml

```diff
@@ -110,3 +110,14 @@ let _ =
  fun aaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbbbbbbb cccccccccccccccccccccccc
      dddddddddddddddddd eeeeeeeeeeeeee ->
   ()
+
+let _ =
+  let (x : int) = x in
+  let x : int = x in
+  let (_ : int) = x in
+  let (_ : int) = x in
+  let%ext (x : int) = x in
+  let%ext x : int = x in
+  let%ext (_ : int) = x in
+  let%ext (_ : int) = x in
```

Comment:
I guess we need a .ref file for some of these to be meaningful.

Reply:
Actually those tests can be removed since they were already present above in the file, and some cases are also tested in extensions.ml.